### DISCUSSION TITLE
middleware/kafka/user-guide/logs/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1899] [Announcements]
# middleware/kafka/user-guide/logs/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/kafka/user-guide/logs/

**Comment by liyuerich** [comment]
是不是需要提前安装 insight-agent？
